db/seeds.rb

```diff
@@ -72,3 +72,9 @@
     MapSegment.create(map_id: map.id, name: segment)
   end
 end
+
+puts "Creating anonymous user"
+User.create(
+  email: "anonymous@ghost.com",
```

Comment:
Ha, I tried visiting ghost.com and it took me to https://www.symantec.com/theme/ghost.